content/en/synthetics/private_locations/_index.md

```diff
@@ -36,7 +38,7 @@ Your private location worker pulls your test configurations from Datadog’s ser
 
 ### Docker
 
-The private location worker is shipped as a Docker container. The official [Docker image][3] is available on Docker Hub. It can run on a Linux based OS or Windows OS if the [Docker engine][4] is available on your host and can run in Linux containers mode.
+Private locations are Docker containers that you can install anywhere inside your private network. You can access the [private location worker image][3] on Docker Hub. It can run on a Linux based OS or Windows OS if the [Docker engine][4] is available on your host and can run in Linux containers mode.
```

Comment:
Did you mean to change "Docker" to "worker" here?

Reply:
Yes! The original link sends users to the worker image on Docker Hub so I added "worker" into the CTA.